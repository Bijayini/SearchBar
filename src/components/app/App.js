import React from 'react';
import Header from '../header/Header';
import SearchBar from '../searchBar/SearchBar';

import('./app.scss');

const App = () => (
  <div>
    <Header />
    <div className="main-container">
      <SearchBar />
    </div>
  </div>
);

export default App;
