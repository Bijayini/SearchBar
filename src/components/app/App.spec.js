import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock('../header/Header', () => () => (
  <div>Header</div>
));
jest.mock('../searchBar/SearchBar', () => () => (
  <div>Search Bar</div>
));

describe("App component", () => {
  it("should render App", () => {
    const {baseElement} = render(<App/>);

    expect(baseElement).toMatchSnapshot();
  });
});
