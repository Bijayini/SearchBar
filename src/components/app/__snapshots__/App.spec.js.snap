// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render App 1`] = `
<body>
  <div>
    <div>
      <div>
        Header
      </div>
      <div
        class="main-container"
      >
        <div>
          Search Bar
        </div>
      </div>
    </div>
  </div>
</body>
`;
